blocks/columns/columns.js

```diff
@@ -7,11 +7,19 @@ export default function decorate(block) {
   // setup image columns
   [...block.children].forEach((row) => {
     [...row.children].forEach((col) => {
-      if (col.textContent.trim() === '') {
+      if (col.firstElementChild === null) {
         col.style.border = 'none';
         col.style.background = 'none';
         col.style.boxShadow = 'none';
       }
+
+      if (col.querySelector('a')?.textContent === 'download') {
+        col.querySelector('a').classList.add('download-btn');
+        const downloadLink = col.querySelector('a');
+        const downloadImg = `<img src="/content/dam/pricefx/style-guide/download-icon.png" alt="download" />`;
```

Comment:
Is that image is not a dynamic one? If not please include the image under `icons` folder and use it. Please check `header.js` for reference.
[blocks/header/header.js](https://github.com/pricefx/pricefx-eds/pull/184/files#diff-cd0c64cd36fcddda1211c38dd4946b01a70443c2d3d0aef4b26ffa8b88e373cfL137)

Reply:
Hi @Vaitheesaran-nagalingam  done with the changes